src/app/app.component.html

```diff
@@ -1,42 +1,22 @@
-
+<!-- Home component -->
 <div class="home-container">
+  <!-- Header Component -->
   <header>
     <div class="nav-header">
       <img class="nav-img" src="../assets/splitwiseLogo.jfif" />
       <div class="log-sign">
-        <div class="nav-log" (click)="navigateLogin()">Login</div>
+        <div class="nav-log" (click)="navigateLogin()">
```

Comment:
You can add buttons in one div. no need to use extra HTML tags if it is not required